images/egress-router/Dockerfile

```diff
@@ -1,8 +1,18 @@
 FROM golang:1.13 as builder
+
```

Comment:
It might be worth using an openshift builder image, so that you can catch problems that are closer to what happens downstream earlier.

Notably, in the downstream build, this does get overwritten by whatever they're doing.

Additionally, if this will be deployed on RHEL7 and RHEL8 systems (maybe it's only RHEL8 systems at this point?) you might need to build binaries for both operating systems, as CNI plugins do run on the host system, and not in a container. Again, example linked dockerfile does have an example of this as well.

For an example, see: https://github.com/openshift/multus-cni/blob/master/Dockerfile.openshift#L2